bot/seasons/pride/pride_avatar.py

```diff
@@ -123,6 +125,19 @@ def crop_ring(ring, px):
 
         await ctx.send(file=file, embed=embed)
 
+    @prideavatar.command()
+    async def flags(self, ctx):
+        """This lists the flags that can be used with the prideavatar command."""
+        choices = sorted(set(OPTIONS.values()))
+        options = "- " + "\n- ".join(choices)
```

Comment:
Hi! Looks good!

Most of our other embeds with lists use `•` instead of `-` as the list character. You can probably just accept the change suggested below:

```suggestion
        options = "• " + "\n• ".join(choices)
```

Should be pretty much ready to go as far as I'm concerned after that.